src/Features/LanguageServer/Protocol/Handler/Symbols/DocumentSymbolsHandler.cs

```diff
@@ -49,54 +46,65 @@ public override async Task<object[]> HandleRequestAsync(DocumentSymbolParams req
                 return Array.Empty<SymbolInformation>();
             }
 
-            var symbols = ArrayBuilder<object>.GetInstance();
-
-            var navBarService = document.Project.LanguageServices.GetRequiredService<Editor.INavigationBarItemService>();
-            var navBarItems = await navBarService.GetItemsAsync(document, cancellationToken).ConfigureAwait(false);
-            if (navBarItems.Count == 0)
+            var navBarService = document.Project.LanguageServices.GetRequiredService<INavigationBarItemService>();
```

Comment:
any concern about this not existing?  for example, for other languages?  not sure the domain invariants here.

Reply:
I believe this is fine, because TS/JS/F# aren't going through our LSP. I'll defer to @dibarbet to be sure though.